### PR TITLE
[milight] Fixed multiple bridge problem

### PR DESCRIPTION
**Pull request by davidgraeff**
Signed-off-by: David Graeff <david.graeff@web.de>

### CODE REVIEW

File: addons/binding/org.openhab.binding.milight/ESH-INF/thing/v3.xml [hunk 1]
```diff
@@ -10,10 +10,9 @@
 			manufacturers.</description>
 
 		<config-description>
-			<parameter name="host" type="text" required="true">
+			<parameter name="host" type="text">
 				<label>IP or Host Name</label>
-				<description>Will be resolved by discovery if auto configured
-				</description>
+				<description>Will be resolved by discovery if auto configured. You either need an IP/Hostename set or the bridgeid.</description>
```
Comment (wborn) [inline addons/binding/org.openhab.binding.milight/ESH-INF/thing/v3.xml]: Hostename -> Hostname
bridgeid -> Bridge ID

File: addons/binding/org.openhab.binding.milight/ESH-INF/thing/v3.xml [hunk 2]
```diff
@@ -39,12 +38,14 @@
...
+			<parameter name="bridgeid" type="text">
 				<label>Bridge ID</label>
 				<description>The mac address of the bridge in upper case letters without delimiter.
+				    Use this and leave the IP/Hostname empty for DHCP environments where IPs may often change over time.
+				    Will be resolved by discovery if auto configured.
+                    
 					This is used to check if the given IP corresponds to the right device. The bridge is set offline if the device
-					does not respond with the correct Bride ID and a re-detection is started. Useful for DHCP environments where
-					IPs may change over time, after power outage etc. Will be resolved by discovery if auto configured.
+					does not respond with the correct Bride ID and a re-detection is started.
```
Comment (wborn) [inline addons/binding/org.openhab.binding.milight/ESH-INF/thing/v3.xml]: Bride ID -> Bridge ID

**Comment by davidgraeff** [comment]
@wborn Issues addressed. Would be nice to have this for OH 2.4.

File: addons/binding/org.openhab.binding.milight/src/main/java/org/openhab/binding/milight/internal/handler/BridgeV6Handler.java [hunk 3]
```diff
@@ -37,6 +39,8 @@
 public class BridgeV6Handler extends AbstractBridgeHandler implements ISessionState {
     private @NonNullByDefault({}) MilightV6SessionManager session;
     final DateTimeFormatter timeFormat = DateTimeFormatter.ofLocalizedTime(FormatStyle.SHORT);
+    private String offlineReasion = "";
```
Comment (wborn) [inline addons/binding/org.openhab.binding.milight/src/main/java/org/openhab/binding/milight/internal/handler/BridgeV6Handler.java]: Thanks for fixing the typos!
There seems to be another one here:
offlineReasion -> offlineReason

**Comment by martinvw** [comment]
@davidgraeff AFAIK this PR is waiting for you?

**Comment by davidgraeff** [comment]
@martinvw I know, but there is an "emergency" fix for mqtt also waiting for me and I can't fix the field name in the webeditor

**Comment by martinvw** [comment]
> @martinvw I know, but there is an "emergency" fix for mqtt also waiting for me and I can't fix the field name in the webeditor

Very good, just wanted to make sure you are aware of it :-)

Thanks for your hard work!

**Comment by wborn** [comment]
I've fixed the last typo and some SAT findings. I think it's good to go now @martinvw !

**Comment by martinvw** [comment]
Do we have more people who can and will test this binding in the snapshot builds?

**Pull request merged**